assets/components/directDebit/directDebitForm/directDebitForm.jsx

```diff
@@ -201,14 +172,16 @@ function ConfirmationInput(props: { checked: boolean, onChange: Function }) {
     <div className="component-direct-debit-form__account-holder-confirmation">
       <div>
         <label htmlFor="confirmation-input">
-          <span>
+          <div className="fancy-checkbox">
```

Comment:
I think a better name for this class, following BEM, would be: `component-direct-debit-form__confirmation-box`. Maybe @Ap0c has an opinion about it.

Reply:
As much as I like the name 'fancy checkbox', I think I agree with @svillafe 😄